src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -959,11 +995,21 @@ def main(argv):
       engine_common.recreate_directory(merge_tmp_dir)
 
     old_corpus_len = shell.get_directory_file_count(corpus_directory)
+    merge_directory = create_merge_directory()
+    corpus_directories.insert(0, merge_directory)
+
+    if use_minijail:
+      target = '/' + MERGE_DIRECTORY_NAME
```

Comment:
Please use the helper 
bind_corpus_dirs(minijail_chroot, [merge_directory])

we use this in similar way for e.g. new_testcase_mutations_directory.